File: PUSGSProjekat2/ProjekatPUSGS/ProjekatPUSGS/Models/BrzaRezervacijaLetova.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace ProjekatPUSGS.Models
{
    public class BrzaRezervacijaLetova
    {
        [Key]
        public int IdRez { get; set; }
        public int IdAvioKompanije { get; set; }
        public int IdLeta { get; set; }
        public double NovaCena { get; set; }
        public double PocetnaCena { get; set; }
        public double Popust { get; set; }
        public bool Zavrseno { get; set; }
        public string IdKlijenta { get; set; }
        public DateTime PocetniDatum { get; set;}
        public DateTime KrajnjiDatum { get; set; }

        [Timestamp]
        public byte[] RedVerzija { get; set; }

    }
}

File: PUSGSProjekat/src/app/entities/brza-rezervacija-letova/brza-rezervacija-letova.ts
import {NumberValueAccessor} from '@angular/forms';

export class BrzaRezervacijaLetova {
    idRez:number;
    idAvioKompanije:number;
    idLeta:number;
    idKlijenta:string;
    novaCena:number;
    pocetnaCena:number;
    popust:number;
    pocetniDatum:string;
    krajnjiDatum:string;
    redVerzija:any[];

    constructor(idRez:number, idAvioKompanije:number, idLeta:number, idKlijenta:string, novaCena:number, pocetnaCena:number, popust:number, pocetniDatum:string, krajnjiDatum:string, redVerzija:any[]) {
        this.idRez=idRez;
        this.idAvioKompanije=idAvioKompanije;
        this.idLeta=idLeta;
        this.idKlijenta=idKlijenta;
        this.novaCena=novaCena;
        this.pocetnaCena=pocetnaCena;
        this.popust=popust;
        this.pocetniDatum=pocetniDatum;
        this.krajnjiDatum=krajnjiDatum;
        this.redVerzija=redVerzija;
    }
}